automation/jenkins_build.sh

```diff
@@ -11,36 +11,82 @@ docker pull resin/${ARCH}-supervisor:${ESCAPED_BRANCH_NAME} || docker pull resin
 # Test the gosuper
 make SUPERVISOR_VERSION=${VERSION} JOB_NAME=${JOB_NAME} test-gosuper
 
-# Build the images
-make SUPERVISOR_VERSION=${ESCAPED_BRANCH_NAME} \
-	ARCH=${ARCH} \
-	JOB_NAME=${JOB_NAME} \
-	DEPLOY_REGISTRY= \
-	PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
-	PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
-	MIXPANEL_TOKEN=${MIXPANEL_TOKEN} \
-	deploy
-make SUPERVISOR_VERSION=${VERSION} \
-	ARCH=${ARCH} \
-	JOB_NAME=${JOB_NAME} \
-	DEPLOY_REGISTRY= \
-	PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
-	PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
-	MIXPANEL_TOKEN=${MIXPANEL_TOKEN} \
-	deploy
-make SUPERVISOR_VERSION=${ESCAPED_BRANCH_NAME} \
-	ARCH=${ARCH} \
-	JOB_NAME=${JOB_NAME} \
-	DEPLOY_REGISTRY=registry.resinstaging.io/ \
-	PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
-	PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
-	MIXPANEL_TOKEN=${MIXPANEL_TOKEN} \
-	deploy
-make SUPERVISOR_VERSION=${VERSION} \
-	ARCH=${ARCH} \
-	JOB_NAME=${JOB_NAME} \
-	DEPLOY_REGISTRY=registry.resinstaging.io/ \
-	PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
-	PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
-	MIXPANEL_TOKEN=${MIXPANEL_TOKEN} \
-	deploy
+if [ -z "$BASE_DISTRO" ]; then
+	# Debian
+	# Build the images
+	make SUPERVISOR_VERSION=${ESCAPED_BRANCH_NAME} \
```

Comment:
Can you do:
```bash
MAKE_ARGS="ARCH=${ARCH} \
	JOB_NAME=${JOB_NAME} \
	BASE_DISTRO= \
	PUBNUB_SUBSCRIBE_KEY=${PUBNUB_SUBSCRIBE_KEY} \
	PUBNUB_PUBLISH_KEY=${PUBNUB_PUBLISH_KEY} \
	MIXPANEL_TOKEN=${MIXPANEL_TOKEN}"
```
and then just
```bash
make ${MAKE_ARGS} \
	SUPERVISOR_VERSION=${ESCAPED_BRANCH_NAME} \
	DEPLOY_REGISTRY= \
	deploy
```
to make it easier to see the difference between the different commands (and reduce duplication).